composer.json

```diff
@@ -65,6 +68,7 @@
         "friendsofsymfony/rest-bundle": "<2.6",
         "jms/serializer": "<0.13",
         "liip/imagine-bundle": "<1.9",
+        "nyholm/psr7": "^1.2",
```

Comment:
why this conflict?

Reply:
Must be a merge conflict. I reverted this